### PR TITLE
Added minimal set of tests for python VisionIpc

### PR DESCRIPTION
**Pull request by mitchellgoffpc**

### CODE REVIEW

File: visionipc/tests/test_visionipc.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  if "ZMQ" in os.environ:
+    time.sleep(t)
+
+
+class TestVisionIpc(unittest.TestCase):
+
+  def tearDown(self):
+    if hasattr(self, 'server'):
+      del self.server
+      gc.collect()
```
Comment (adeebshihadeh) [inline visionipc/tests/test_visionipc.py:19]: why is this necessary?

Comment (mitchellgoffpc) [reply visionipc/tests/test_visionipc.py:19]: I was seeing occasional `Error, failed to bind PubSocket to 9000: Address already in use` errors in CI, wanted to check if this fixed it
